issues/quay-mesos.md

```diff
@@ -4,9 +4,6 @@ sublayout: issue
 title: Pulling Private Quay.io images with Marathon/Mesos fails
 frontpage: True
 ---
-
-#### Special thanks to [Ian Saunders](https://github.com/IanSaunders) who provided this solution.
```

Comment:
The first line is always a description of the problem and if we end up getting other solutions not from him, it doesn't make sense to have his name at the top of the article, but in the section that he actually contributed. You said this was okay when I asked yesterday.